prime-router/src/main/kotlin/ValueSet.kt

```diff
@@ -28,6 +30,7 @@ data class ValueSet(
         NULLFL
     }
 
+    @get:JsonIgnore
     val systemCode
         get() = when (system) {
             SetSystem.HL7 -> name.uppercase()
```

Comment:
This was a fun bug. ValueSets didn't serialize nicely to JSON because of this property. I had to mark it as ignore.

Reply:
I don't know the expected functionality around here - when we are serializing this to JSON we are not going to include systemCode it looks like. Will this have any negative consequences down the line when deserializing?